smart_contracts.rst

```diff
@@ -165,51 +168,38 @@ Allows a channel participant to close the channel. The channel cannot be settled
 
     Only a valid signed balance proof from the channel partner (the other channel participant) must be accepted. This balance proof sets the amount of tokens owed to the participant by the channel partner.
 
-**Update transfer state**
+**Update non-closing participant balance proof**
 
-Called after a channel has been closed. Allows the non-closing participant to provide a balance proof for the latest transfer from the closing participant. This modifies the state for the closing participant.
+Called after a channel has been closed. Can be called by any Ethereum address and allows the non-closing participant to provide the latest balance proof from the closing participant. This modifies the stored state for the closing participant.
 
 ::
 
-    function updateTransfer(
+    function updateNonClosingBalanceProof(
         uint channel_identifier,
+        bytes32 balance_hash,
```

Comment:
what about the monitoring services priority? are we going to add it? is it not the goal of this PR?

Reply:
> so only handle the MS priorities in the Raiden client.

we can configure the monitoring services priorities offchain, but we need to enforce it on-chain otherwise there is little point in having a priority field.

Reply:
Opened https://github.com/raiden-network/spec/issues/58 for this. This will be included in the Monitoring Service smart contract. So it will be external to the core contracts anyway.